content/news: Add post for April Fools Day 2024 (#342)
mixxxdj/website: content/news/XXXX-XX-XX-mixxx-plans-to-acquire-alphatheta-serato-ni.md

title: Mixxx Plans to Acquire AlphaTheta, Serato and Native Instruments
authors: Jan Holthuis
tags: takeover, super important business decision
status: draft
comments: yes

The Mixxx Board of Directors in pleased to announce plans to acquire the AlphaTheta Corporation (formerly known as Pioneer DJ), Serato Audio Research Ltd. and Native Instruments.

> *"This exciting consolidation of brands will reshape the DJ software market as we know it"*, Mixxx CEO Scrooge McScratch said after the decision was made.

Author: ```suggestion
*"This exciting [consolidation of brands](https://www.thepost.co.nz/a/business/350059653/us-firm-waves-prospect-legal-action-stop-100m-buy-out-nz-software-firm-serato) will reshape the DJ software market as we know it"*, Mixxx CEO Scrooge McScratch said after the decision was made.
```
@Swiftb0y like this?

Some trademarks such as Rekordbox®, Serato® and Traktor® will likely be dropped to archieve a lean portfolio and focus on the popular Mixxx brand.

The move will lead to new synergies and enables Mixxx to benefit from the know-how of its former competitors.

> *"Our new subsidiaries have a track record in profitable partnerships with hardware manufacturers."*, McStrach commented.

Reviewer: ```suggestion
*"Our new subsidiaries have a track record in profitable partnerships with hardware manufacturers."*, McScratch commented.
```

> *"We want to leverage their expertise to lock down DJ hardware and ensure that the majority of DJ equipment in the market will only work with Mixxx. We are committed to our customers, so it's only fair to demand that they stay committed to us as well."*

Author: ```suggestion
*"We want to leverage their expertise on locking down DJ hardware artificially and ensure that the majority of DJ equipment in the market will only work with Mixxx. We are committed to our customers, so it's only fair to demand that they stay committed to us as well."*
```
